client/src/language/EmbeddedLanguageDocsManager.ts

```diff
@@ -182,28 +160,6 @@ export default class EmbeddedLanguageDocsManager {
       await this.createEmbeddedLanguageDocFile(embeddedLanguageDoc)
     }
   }
-
-  async deleteEmbeddedLanguageDocs (originalUriString: string): Promise<void> {
```

Comment:
So what's the status now with deleting "virtual" embedded docs? Do we remove them at some point?
Is it always growing in size and the user will have to manually clean up?
The cpptools does this and it's annoying. I think we can easily implement a way to do some garbage cleaning if we're not already.

Reply:
I thought we were doing like cpptools. If these is not what we want to do, then one alternative could be to delete the "embedded-languages" folder on "deactivate".

Reply:
I'd say this makes at least two improvements that could be done with garbage collecting:
- Clean up "embedded-languages" folder once in while
- Handle disposables properly (there are several of them in languageClient.ts that are never cleaned up)

Reply:
Ok, don't forget to create a ticket on Redmine to keep track of this.